packages/server-boilerplate/src/microService/utils/logApiRequest.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { NextFunction, Request, Response } from 'express';
+import { ServerBoilerplateModelRegistry } from '../../types';
+
+export const logApiRequest = (
```

Comment:
These functions are a little different between microservice and orchestration servers, so I created two separate ones